api/v1alpha1/physical_machine_chaos_types.go

```diff
@@ -332,6 +337,13 @@ type NetworkPartitionSpec struct {
 	AcceptTCPFlags string `json:"accept-tcp-flags,omitempty"`
 }
 
+type NetworkDownSpec struct {
+	// the network interface to impact
+	Device string `json:"device,omitempty"`
+	// NIC down time, time units: ns, us (or µs), ms, s, m, h.
+	Duration string `json:"duration,omitempty"`
```

Comment:
There is another `Duration` in `PhysicalMachineChaosSpec `, I think it's better to resue this one
https://github.com/chaos-mesh/chaos-mesh/blob/8f10165d47dac0f917b4527bb4e8abbf0d0a6db3/api/v1alpha1/physical_machine_chaos_types.go#L78

Reply:
We can keep the `Duration` here, and set it to the value of `PhysicalMachineChaosSpec.Duration` in the webhook.